### DISCUSSION TITLE
Overridden functionality for the ROW component breaking side effects for pagination and search

### DISCUSSION THREAD

**Discussion by islandnuge** [Power-Components/livewire-powergrid#871] [Q&A]
I had to override the local implementation of the Row component (`resources/views/vendor/livewire-powergrid/components/row.blade.php`) after publishing the views but this has somehow had an adverse affect on the pagination, filtering and searching functionality.

Is there any dependencies on these functions that I need to re-wire in order for these to work? I don't think i've materially changed anything else and my config remained the same before the row edits.

Any suggestions are welcomed!

**Comment by luanfreitasdev** [comment]
Hello!. Unfortunately some of these files are essential for the functioning of the table
